pkg/reading/service_read_issue.go

```diff
@@ -71,3 +82,25 @@ func (s *ServiceReadIssue) PostAPICall(result interface{}) error {
 func (s *ServiceReadIssue) Name() string {
 	return "ServiceReadIssue"
 }
+func (s *ServiceReadIssue) ExecuteAsLastStep(ctx configuration.Context) error {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [exported method ServiceReadIssue.ExecuteAsLastStep should have comment or be unexported](https://app.codacy.com/manual/TurnsCoffeeIntoScripts/jira-api-resource/pullRequest?prid=4860757)